addons/web/static/src/core/tags_list/tags_list.xml

```diff
@@ -18,7 +18,7 @@
 
                 <!-- Avatar's :hover backdrop -->
                 <span
-                    t-if="tag.img &amp;&amp; props.displayText"
+                    t-if="tag.img &amp;&amp; props.displayText &amp;&amp; !props.readonly"
```

Comment:
I would use tag.onDelete instead of introducing a new props

Reply:
also, `&amp;&amp;` can be `and`

Reply:
and you should add a test

Reply:
@aab-odoo thanks for the review, I replaced the && and the props with the onDelete. However as a designer this task was a bit out of my scope even if I enjoy Js and learning OWL, I still have no clue how to write a proper QUnit test for now, if you could handle it I would be grateful 🙏